### PR TITLE
re-open e2e-test for helm schematic

### PR DESCRIPTION
**Pull request by captainroy-hy**
as title.
Signed-off-by: roy wang <seiwy2010@gmail.com>

### CODE REVIEW

File: pkg/controller/core.oam.dev/v1alpha2/applicationcontext/applicationcontext_controller.go [hunk 1]
```diff
@@ -83,7 +82,9 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 	}
 	appConfig.Status = appContext.Status
 	// the name of the appConfig has to be the same as the appContext
-	appConfig.ObjectMeta = metav1.ObjectMeta{Namespace: appContext.Namespace, Name: appContext.Name, UID: appContext.UID}
```
Comment (captainroy-hy) [inline pkg/controller/core.oam.dev/v1alpha2/applicationcontext/applicationcontext_controller.go:86]: AC controller needs ownerRef to determine isControlledByApp. So we should not override ownerRef here.

**Comment by wonderflow** [comment]
Does it the same with #1261 ? Can you include changes in that one

**Comment by captainroy-hy** [comment]
> Does it the same with #1261 ? Can you include changes in that one

Not exactly same. After offline discussion, @yangsoon will include his change after the helm e2e issue fixed.

**Comment by wonderflow** [comment]
Cool， please fix the conflicts @

> > Does it the same with #1261 ? Can you include changes in that one
> 
> Not exactly same. After offline discussion, @yangsoon will include his change after the helm e2e issue fixed.

**Review by wonderflow — APPROVED** [review]
Cool job!

**Pull request merged**